packages/calcite-components/src/utils/floating-ui.spec.ts

```diff
@@ -80,9 +80,9 @@ describe("repositioning", () => {
     expect(floatingEl.style.left).toBe("0");
   }
 
-  it("repositions only for open components", async () => {
+  it("repositions for unopened components", async () => {
     await reposition(fakeFloatingUiComponent, positionOptions);
```

Comment:
For clarity, could you call `assertPreOpenPositioning` at the beginning of each of these tests?